src/constants/index.ts

```diff
@@ -26,8 +26,8 @@ export const RANGO_API_KEY = '6165008b-282e-446f-acf8-13c2b06d0775';
 // one basis point
 export const ONE_BIPS = new Percent(JSBI.BigInt(1), JSBI.BigInt(10000));
 
-// used to ensure the user doesn't send so much ETH so they end up with <.01
-export const MIN_ETH: JSBI = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(16)); // .01 ETH
+// used to ensure the user doesn't send so much ETH so they end up with <.000001
+export const MIN_ETH: JSBI = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(12)); // .000001 ETH
```

Comment:
@prodesert22 Looks like attached issue is for Skale testnet but this is global change. Are we sure that this is not going to affect any other chain ?

Reply:
Will affect other chains, but I think is minor change because the user decide if he will send 0.000001 or we need it for tokens with low liquidity where it is only possible to exchange these small values